src/main/java/org/gridsuite/sensitivityanalysis/server/entities/SensitivityEmbeddable.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.sensitivityanalysis.server.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author Laurent Garnier <laurent.garnier at rte-france.com>
+ */
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Embeddable
+public class SensitivityEmbeddable {
+
+    @Column
+    private int factorIndex;
+
+    @Column
+    private int contingencyIndex;
+
+    @Column(name = "value_")
```

Comment:
what's the reason to add "_" for col name ?

Reply:
I had to check to find back that I just followed the changes that occurred for migration to SpringBoot 2.7.
It seems that the name "value" for a column is to be avoided